crates/aptos/src/common/types.rs

```diff
@@ -844,14 +939,47 @@ impl PrivateKeyInputOptions {
     }
 }
 
+// Extract the public key by derivating private key, fall back to public key from profile
+// Order of operations
+// 1. Get the private key (either from CLI input or profile), and derive the public key problem private key
```

Comment:
```suggestion
// 1. Get the private key (either from CLI input or profile), and derive the public key from it
```

Not sure if this was the intended sentence, but it's my guess haha

Reply:
Yep you got it!

Not sure what I was thinking when I typed that comment lol

Reply:
Probably just a typo lol